gulpfile.js

```diff
@@ -11,9 +11,6 @@ gulp.task('minify-icons', function() {
     .pipe(svgmin({
       plugins: [
         {
-          removeDimensions: true
```

Comment:
Not positive, but I think it might. I want to wait until after launch to mess with it though.